src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLAlertImpl.java

```diff
@@ -95,7 +95,10 @@ public DataFetcher<Integer> alertHash() {
 
   @Override
   public DataFetcher<String> alertHeaderText() {
-    return environment -> getSource(environment).headerText().toString(environment.getLocale());
+    return environment -> {
+      var headerText = getSource(environment).headerText();
+      return headerText != null ? headerText.toString(environment.getLocale()) : null;
```

Comment:
Isn't there `LegacyGraphQLUtils` for this?

Reply:
Indeed, I don't know why I forgot it did exactly this